protocols/v2/noise-sv2/src/cipher_state.rs

```diff
@@ -88,24 +179,45 @@ pub enum GenericCipher {
 }
 
 impl Drop for GenericCipher {
+    // Securely erases the encryption key when the [`GenericCipher`] is dropped.
+    //
+    // Ensures that the encryption key is securely erased from memory when the [`GenericCipher`]
+    // instance is dropped, preventing any potential leakage of sensitive cryptographic material.
     fn drop(&mut self) {
         self.erase_k();
     }
 }
 
 impl GenericCipher {
+    // Encrypts the data (`msg`) in place using the underlying cipher.
```

Comment:
Commented previously on this but didn't get a response. why are we using ( ` X ` ) to refer to variables?

Reply:
Rachel suggested these, cc: @rrybarczyk